core/stores/sqlx/tx.go

```diff
@@ -101,13 +113,16 @@ func (t txSession) QueryRowsPartial(v interface{}, q string, args ...interface{}
 }
 
 func (t txSession) QueryRowsPartialCtx(ctx context.Context, v interface{}, q string,
-	args ...interface{}) error {
-	ctx, span := startSpan(ctx)
-	defer span.End()
+	args ...interface{}) (err error) {
```

Comment:
(err error) to error?

Reply:
we need err, which is used when defer calls.